packages/@dcl/inspector/src/components/Input/Input.tsx

```diff
@@ -17,22 +17,18 @@ const Input = ({ value, onCancel, onSubmit, onChange, placeholder }: PropTypes)
 
     const getValue = () => ref.current?.value || value
 
-    const onBodyClick = (e: MouseEvent) => {
-      if (e.target !== ref.current) onSubmit && onSubmit(getValue())
-    }
-
     const onKeyUp = (e: KeyboardEvent) => {
       if (cancelingKeys.has(e.key)) onCancel && onCancel()
       if (submittingKeys.has(e.key)) onSubmit && onSubmit(getValue())
     }
 
-    const onBlur = (_: Event) => onSubmit && onSubmit(getValue())
+    const onBlur = (_: Event) => {
+      if (blurBehavior == BlurBehavior.CANCEL) onCancel && onCancel()
```

Comment:
what if we add an onBlur property ? 
like
```tsx
<Input onChange={someChangeFn} onCancel={someCancelFn} onBlur={someCancelFn} />
```

Reply:
Done.